trunk/Public/DynamicConditionsPublic.php

```diff
@@ -235,13 +233,34 @@ private function convertAcfDate( $id, array $data ) {
         $this->elementSettings[$id]['dynamicconditions_dynamic'] = $timestamp;
     }
 
+
+    /**
+     * Hooks into elementor/theme/before_do_popup, loops all popups and check for condition
+     * Removes popup from location, if it is hidden
```

Comment:
Does this method "check popups condition" or "remove a/the popup from a/the/it's location if it (the popup or the location?) is hidden"?  
The description and the method name do not really match.